content/announcements/2025-11-26-ingress-nginx-eol-and-gateway-api.md

```diff
@@ -0,0 +1,181 @@
+velocity, the Gateway API design intentionally addresses a security concern with
+the Ingress API: nothing prevents one team from accidentally or maliciously
+capturing traffic intended for another team by creating an Ingress with the same
+hostname but different TLS configuration. This often happens in larger clusters
+with many teams, where conflicting Ingress objects can silently intercept
+traffic meant for other services. By centralizing TLS configuration at the
+Gateway level, Gateway API provides stronger security boundaries, at the cost of
+reduced self-service in simple multi-tenant setups. For more details on the
+design rationale, you can read the page
+[Key differences between Ingress API and Gateway API](https://gateway-api.sigs.k8s.io/guides/migrating-from-ingress/#key-differences-between-ingress-api-and-gateway-api).
+
+## Why cert-manager can't fix this on its own (yet)
+
+cert-manager's current Gateway API integration only supports TLS configuration
+defined on the Gateway itself: cert-manager watches Gateway resources. If the
+`cert-manager.io/issuer` or `cert-manager.io/cluster-issuer` annotation is
+present on one of the the Gateway resources, and given that there is a listener
+that uses the HTTPS protocol and that the field `tls.certificateRefs` isn't
+empty, cert-manager will create a Certificate, and will create a Secret with the
+name set in `certificateRefs`.
+
+Here is what the Gateway must look like for cert-manager to pick it up:
+
+```yaml
+apiVersion: gateway.networking.k8s.io/v1
+kind: Gateway
+metadata:
+  name: istio-gateway
+  annotations:
+    cert-manager.io/cluster-issuer: letsencrypt-prod
+spec:
+  gatewayClassName: istio
+  listeners:
+    - name: https
+      hostname: 'foo.example.com'
+      port: 443
+      protocol: HTTPS
+      allowedRoutes:
+        namespaces:
+          from: All
+      tls:
+        mode: Terminate
+        certificateRefs:
+          - name: gateway-tls
+```
+
+For context, cert-manager doesn't look at the hostnames on HTTPRoutes as these
+hostnames aren't meant for TLS; they are used by Gateway API controllers to know
+which listener on the Gateway should be used for each HTTPRoute.
+
+This model works only when either:
+
+1. each team owns its own Gateway (increasing cost/infra), or
+2. the implementation makes Gateways "cheap" logical objects.
+
+It does not work for the common "one-shared-Gateway + many-teams" model. There
+is no safe way today to let each team self-manage TLS, on a shared
+Gateway, without either risky wildcards or dangerous RBAC.
+
+## ListenerSet: the missing building block
+
+Gateway API intends to introduce the ListenerSet resource (currently only as the
+experimental `XListenerSet` kind) to address the self-service challenge while
+maintaining security boundaries. ListenerSet allows:
+
+- the platform team to own one shared Gateway and its infrastructure,
+- application teams to create their own listeners and TLS settings in
+  `XListenerSet` objects.
```

Comment:
```suggestion
- The platform team to own one shared Gateway and its infrastructure.
- Each application team can create their own listeners and TLS settings in
  `XListenerSet` objects.
```

I'm not a huge fan of trying to string sentences across bullets/numbers, it usually goes poorly.